orchagent/copporch.cpp

```diff
@@ -75,12 +77,68 @@ std::map<string, sai_packet_action_t> packet_action_map = {
     {"transit", SAI_PACKET_ACTION_TRANSIT}
 };
 
+std::vector<sai_hostif_trap_id_t> default_trap_ids = {
+    SAI_HOSTIF_TRAP_ID_TTL_ERROR
+};
+
+const string default_trap_group = "default";
+
 CoppOrch::CoppOrch(DBConnector *db, string tableName) :
     Orch(db, tableName)
 {
     SWSS_LOG_ENTER();
+    initDefaultTrapGroup();
+    initDefaultTrapIds(default_trap_ids);
 };
 
+void CoppOrch::initDefaultTrapIds(std::vector<sai_hostif_trap_id_t> &trap_ids)
```

Comment:
where is this input trap_ids used? Just put the initialization code in constructor here, as the functions are not too long and it only used once here.